test/e2e/baseline_test.go

```diff
@@ -1,6 +1,3 @@
-//go:build integ
-// +build integ
```

Comment:
cc @YaoZengzeng why did we need this tag

Reply:
It is used to seperate test files, we will use it in: https://github.com/kmesh-net/kmesh/blob/main/test/e2e/run_test.sh#L278 .

Although all the files in `./test/e2e/` are used to e2e test.

Reply:
Will revert